samples/SkiaMultiplatformSample/src/commonMain/kotlin/org/jetbrains/skiko/sample/Clocks.kt

```diff
@@ -11,7 +11,7 @@ import kotlin.math.sin
 import kotlin.math.PI
 import kotlin.math.pow
 
-class Clocks(private val layer: SkiaLayer): SkikoView {
+open class Clocks(private val layer: SkiaLayer): SkikoView {
```

Comment:
Why this change?

Reply:
Temporarily opened to add custom key binding for macOS